### PR TITLE
docs(core): Translated the readme to French and German

### PR DESCRIPTION
**Pull request by VanshajPoonia**
This Pull Request solves issues #2615 and #2616 

- I have translated the readme file into French and German.
- Added the functional links to the Japanese readme in all the other readme files as well.
- Naming of the readme done as per the guidelines.

### CODE REVIEW

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: i18n/README-PT.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 </p>
 
 
-[English](https://github.com/questdb/questdb) | [简体中文](README.zh-cn.md) | [繁體中文](README.zh-hk.md) | [العربية](README.ar-dz.md) | [Italiano](README.it-it.md) | [Українська](README.ua-ua.md) | [Español](README.es-es.md) | Português | [日本](./README.ja-ja.md)
+[English](https://github.com/questdb/questdb) | [简体中文](README.zh-cn.md) | [繁體中文](README.zh-hk.md) | [العربية](README.ar-dz.md) | [Italiano](README.it-it.md) | [Українська](README.ua-ua.md) | [Español](README.es-es.md) | Português | [日本](./README.ja-ja.md) | [Français](./README.fr-fr.md) | [Deutsch](./README.du-du.md)
```
[the remark this answers is not in the document]
Comment (VanshajPoonia) [reply i18n/README-PT.md]: I'm sorry! Actually, I didn't check the code and wrote it the way I pronounce it. I have updated the files accordingly.

**Comment by bluestreak01** [comment]
Are you a native German and French speaker?

**Comment by VanshajPoonia** [comment]
> Are you a native German and French speaker?

I am not a native speaker. I've learnt French in school for 6 years (till class 10th; got 99/100 in the final exams).
I learnt German in 2020 from a YT channel called [Learn German with GermanPod101.com](https://youtube.com/c/Germanpod101) and also used Dictionary and Google translate for the parts which I wasn't sure about.
I know some other languages too but translations for most langauges that I know are already available.
Currently, I am looking forward to translate other docs too but I have my exams this month.

**Comment by bluestreak01** [comment]
thanks a lot! Our native French and German speakers are away right now. They will check your PR ASAP

File: i18n/README.de-de.md [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+schnelle SQL-Abfragen mit einfacher Bedienung. Es unterstützt Schema-Agnostik
+Aufnahme mit dem InfluxDB-Zeilenprotokoll, dem PostgreSQL-Drahtprotokoll und einem REST
+API für Massenimporte und -exporte.
+
+QuestDB ist gut geeignet für Finanzmarktdaten, Anwendungsmetriken, Sensoren
+Daten, Echtzeitanalysen, Dashboards und Infrastrukturüberwachung.
+
+QuestDB implementiert ANSI SQL mit nativer Zeitreihen-SQL-Semantik. Diese SQL
+Semantik macht es einfach, Daten aus mehreren Quellen zu korrelieren
+relationale und Zeitreihen-Joins. Wir erreichen eine hohe Leistung, indem wir a
```
Comment (sandroqdb) [inline i18n/README.de-de.md:32]:
```suggestion
relationalen und Timeseries-Joins. Wir erreichen eine hohe Performance durch
```

File: i18n/README.de-de.md [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+spaltenorientiertes Speichermodell, parallelisierte Vektorausführung, SIMD-Anweisungen,
+und Low-Latency-Techniken. Die gesamte Codebasis ist von Grund auf neu aufgebaut
+Java und C++, ohne Abhängigkeiten und ohne Garbage Collection.
```
Comment (sandroqdb) [inline i18n/README.de-de.md:35]:
```suggestion
in Java und C++, ohne Abhängigkeiten und ohne Garbage Collection.
```

File: i18n/README.de-de.md [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+
+<div align="center">
+  <a href="https://demo.questdb.io">
+    <img alt="Die QuestDB-Webkonsole zeigt mehrere SQL-Anweisungen und visualisiert eine Abfrage als Diagramm" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="600" />
+  </a>
+</div>
+
+## Probieren Sie QuestDB aus
+
+Wir stellen eine [Live-Demo](https://demo.questdb.io/) bereit, die mit den neuesten bereitgestellt wird
```
Comment (sandroqdb) [inline i18n/README.de-de.md:45]:
```suggestion
Wir stellen eine [Live-Demo](https://demo.questdb.io/) bereit, die mit den neuesten QuestDB Release
```

File: i18n/README.de-de.md [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+QuestDB-Release und Beispieldatensätze:
```
Comment (sandroqdb) [inline i18n/README.de-de.md:46]:
```suggestion
und Beispieldatensätze bereitgestellt wird:
```

File: i18n/README.de-de.md [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+```bash
+brew install questdb
+brew services start questdb
+
+questdb start // To start questdb
+questdb stop  // To stop questdb
+```
+
+Die [QuestDB-Downloadseite](https://questdb.io/get-questdb/) bietet direkte
+Downloads für Binärdateien und enthält Details für andere Installationen und Bereitstellungen
```
Comment (sandroqdb) [inline i18n/README.de-de.md:73]:
```suggestion
Downloads für Binärdateien und enthält Details für andere Installationen und
```

File: i18n/README.de-de.md [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+Methoden.
+
+### Verbinden Sie sich mit QuestDB
+
+Sie können mit QuestDB über die folgenden Schnittstellen interagieren:
+
+- [Web Console](https://questdb.io/docs/develop/web-console/) für interaktiv
+  SQL-Editor auf Port `9000`
+- [InfluxDB line protocol](https://questdb.io/docs/reference/api/influxdb/) zum
```
Comment (sandroqdb) [inline i18n/README.de-de.md:82]:
```suggestion
- [InfluxDB line protocol](https://questdb.io/docs/reference/api/influxdb/) zur
```

File: i18n/README.de-de.md [hunk 8]
```diff
@@ -0,0 +1,326 @@
...
+  Aufnahme mit hohem Durchsatz am Port `9009`
+- [REST API](https://questdb.io/docs/reference/api/rest/) am port `9000`
+- [PostgreSQL wire protocol](https://questdb.io/docs/reference/api/postgres/) am
+  port `8812`
+
+### Daten einfügen
+
+Nachfolgend finden Sie unsere offiziellen InfluxDB-Leitungsprotokollclients für beliebte Programmierung
```
Comment (sandroqdb) [inline i18n/README.de-de.md:90]:
```suggestion
Nachfolgend finden Sie unsere offiziellen InfluxDB-Leitungsprotokollclients für beliebte Programmier-
```

File: i18n/README.de-de.md [hunk 9]
```diff
@@ -0,0 +1,326 @@
...
+- [Go](https://pkg.go.dev/github.com/questdb/go-questdb-client)
+- [Java](https://questdb.io/docs/reference/clients/java_ilp/)
+- [NodeJS](https://questdb.github.io/nodejs-questdb-client)
+- [Python](https://py-questdb-client.readthedocs.io/en/latest/)
+- [Rust](https://docs.rs/crate/questdb-rs/latest)
+
+## Vergleich von QuestDB mit anderen Open-Source-TSDBs
+
+[Dieser Artikel](https://questdb.io/blog/2021/07/05/comparing-questdb-timescaledb-influxdb/)
+vergleicht QuestDB übergreifend mit anderen Open-Source-Zeitreihendatenbanken
```
Comment (sandroqdb) [inline i18n/README.de-de.md:104]:
```suggestion
vergleicht QuestDB übergreifend mit anderen Open-Source-Timeseriesdatenbanken
```

File: i18n/README.de-de.md [hunk 10]
```diff
@@ -0,0 +1,326 @@
...
+[Zeitreihen-Benchmark-Suite](https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/)
+Ergebnisse unter Verwendung des Anwendungsfalls "nur CPU" mit 6 Arbeitern auf einem AMD Ryzen 3970X:
+
+<div align="center">
+  <a href="https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/">
+    <img alt="Ein Diagramm, das den maximalen Durchsatz von QuestDB, ClickHouse, TimescaleDB und InfluxDB vergleicht." src=".github/tsbs-results.png"/>
+  </a>
+</div>
+
+Die folgende Tabelle zeigt die Abfrageausführungszeit von einer Milliarde Zeilen, die auf a ausgeführt werden
```
Comment (sandroqdb) [inline i18n/README.de-de.md:117]:
```suggestion
Die folgende Tabelle zeigt die Abfrageausführungszeit von einer Milliarde Zeilen, die auf einer
```

File: i18n/README.de-de.md [hunk 11]
```diff
@@ -0,0 +1,326 @@
...
+`c5.metal`-Instanz mit 16 der 96 verfügbaren Threads:
```
Comment (sandroqdb) [inline i18n/README.de-de.md:118]:
```suggestion
`c5.metal`-Instanz mit 16 der 96 verfügbaren Threads ausgefuehrt werden:
```

File: i18n/README.de-de.md [hunk 12]
```diff
@@ -0,0 +1,326 @@
...
+
+## Ressourcen
+
+### 📚 Lesen Sie die Dokumente
+
+- [QuestDB-Dokumentation:](https://questdb.io/docs/introduction/) verstehen, wie
+  zum Ausführen und Konfigurieren von QuestDB.
+- [Tutorials:](https://questdb.io/tutorial/) Erfahren Sie, was mit QuestDB möglich ist
+  Schritt für Schritt.
+- [Produkt-Roadmap:](https://github.com/questdb/questdb/projects) sehen Sie sich unsere an
```
Comment (sandroqdb) [inline i18n/README.de-de.md:134]:
```suggestion
- [Produkt-Roadmap:](https://github.com/questdb/questdb/projects) sehen Sie sich unsere Roadmap an
```

File: i18n/README.de-de.md [hunk 13]
```diff
@@ -0,0 +1,326 @@
...
+  Planen Sie für kommende Veröffentlichungen.
+
+### ❓ Holen Sie sich Unterstützung
+
+- [Community Slack:](https://slack.questdb.io) Nehmen Sie an technischen Diskussionen teil, fragen Sie
+  Fragen und treffen Sie andere Benutzer!
+- [GitHub-Probleme:](https://github.com/questdb/questdb/issues) Fehler melden oder
+  Probleme mit QuestDB.
+- [Stack Overflow:](https://stackoverflow.com/questions/tagged/questdb) suchen
```
Comment (sandroqdb) [inline i18n/README.de-de.md:143]:
```suggestion
- [Stack Overflow:](https://stackoverflow.com/questions/tagged/questdb) suchen Sie
```

**Review by sandroqdb — CHANGES_REQUESTED** [review]

File: i18n/README-PT.md (hunk 1 above)
Comment (sandroqdb) [inline i18n/README-PT.md]:
```suggestion
[English](https://github.com/questdb/questdb) | [简体中文](README.zh-cn.md) | [繁體中文](README.zh-hk.md) | [العربية](README.ar-dz.md) | [Italiano](README.it-it.md) | [Українська](README.ua-ua.md) | [Español](README.es-es.md) | Português | [日本](./README.ja-ja.md) | [Français](./README.fr-fr.md) | [Deutsch](./README.de-de.md)
```

**Review by sandroqdb — CHANGES_REQUESTED** [review]

File: i18n/README-PT.md (hunk 1 above)
Comment (VanshajPoonia) [reply i18n/README-PT.md]: I think I have already done this change

**Comment by bluestreak01** [comment]
would you mind resolving conflicts pls ?

**Comment by VanshajPoonia** [comment]
> would you mind resolving conflicts pls ?

I'm sorry but I couldn't get which conflicts you are discussing.
I have committed to the suggestions that were made and visible to me.
I request you to please guide me a little

File: i18n/README.de-de.md [hunk 14]
```diff
@@ -0,0 +1,326 @@
...
+- [Rust](https://docs.rs/crate/questdb-rs/latest)
+
+## Vergleich von QuestDB mit anderen Open-Source-TSDBs
+
+[Dieser Artikel](https://questdb.io/blog/2021/07/05/comparing-questdb-timescaledb-influxdb/)
+vergleicht QuestDB übergreifend mit anderen Open-Source-Timeseriesdatenbanken
+Funktionalität, Reife und Leistung.
+
+Hier sind hohe Kardinalität
+[Zeitreihen-Benchmark-Suite](https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/)
```
Comment (sandroqdb) [inline i18n/README.de-de.md:108]:
```suggestion
[Timeseries-Benchmark-Suite](https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/)
```

**Review by sandroqdb — CHANGES_REQUESTED** [review]
Requested 1 more change in row 108.

Besides that, the German translation looks good.

File: i18n/README.de-de.md (hunk 14 above)
Comment (sandroqdb) [reply i18n/README.de-de.md:108]: Requested one more change for row 108. Besides that, the German translation looks good.

**Comment by bluestreak01** [comment]
there is no interest from @tris0laris

**Pull request closed without merging**